.github/workflows/main_ci.yml

```diff
@@ -2,7 +2,7 @@ name: Main CI
 
 on:
   push:
-    branches: [main]
+    branches: [main, bugfix/fix-issue-145-compile-on-msvc-2022]
```

Comment:
I'm assuming you just intend this to be temporary for testing?

Reply:
absolutely